AirCasting/Settings/BackendSettingsView.swift

```diff
@@ -14,6 +14,7 @@ struct BackendSettingsView: View {
     @State private var portText: String = ""
     @State private var url: URL?
     @State private var alertPresented = false
+    @Binding var isShowingLoadingScreen: Bool
```

Comment:
shouldn't we add initial value here instead of having to pass it in via constructor?

Reply:
Yes, thank you for all the comments, changing everything now 🚀